src/app/portlets/dot-edit-page/content/components/dot-form-selector/dot-form-selector.component.ts

```diff
@@ -39,8 +56,17 @@ export class DotFormSelectorComponent implements OnInit {
                 this.messages = messages;
             });
 
+        this.paginatorService.paginationPerPage = 5;
         this.paginatorService.url = 'v1/contenttype?type=FORM';
     }
+    ngOnChanges(changes: SimpleChanges) {
+        setTimeout(() => {
+            if (changes.show.currentValue === true) {
+                this.contentMinHeight =
+                    this.dialog.dialog.nativeElement.getBoundingClientRect().height - 100;
```

Comment:
dialog.dialog?

maybe we should change this one

https://github.com/dotCMS/core-web/pull/1108/files#diff-0db0fb48a9fe56dc260a60ff6b57db62R38

for something like: 'dotDialogComponent'

Reply:
maybe `#dotDialog`